src/TLuaInterpreter.cpp

```diff
@@ -1623,26 +1623,80 @@ int TLuaInterpreter::feedTriggers(lua_State* L)
 {
     Host& host = getHostFromLua(L);
 
-    std::string text;
     if (!lua_isstring(L, 1)) {
         lua_pushfstring(L,
                         "feedTriggers: bad argument #1 type (imitation MUD server text as string\n"
                         "expected, got %s!)",
                         luaL_typename(L, 1));
         return lua_error(L);
-    } else {
-        text = lua_tostring(L, 1);
     }
 
-    QString previousEncoding = host.mTelnet.getEncoding();
-    if (previousEncoding.isEmpty()) {
-        previousEncoding = QStringLiteral("ASCII");
+    bool dataIsUtf8Encoded = false;
+    if (lua_gettop(L) > 1) {
+        if (!lua_isboolean(L, 2)) {
+            lua_pushfstring(L,
+                            "feedTriggers: bad argument #2 type (dataIsUtf8Encoded as boolean {defaults to\n"
```

Comment:
This should be a human-readable name per convention.

Reply:
What would you suggest `isUTF8` or perhaps `convertFromUTF8`...? :thought_balloon:

Reply:
Perhaps @Mudlet/lua-interface can chime in here on this.

Reply:
For the moment I've gone for `Utf8Encoded`...